apps/citibikebymk/citibike_by_mk.star

```diff
@@ -0,0 +1,97 @@
+load("encoding/base64.star", "base64")
+load("encoding/json.star", "json")
+load("http.star", "http")
+load("render.star", "render")
+load("schema.star", "schema")
+
```

Comment:
Code executing at this level (as opposed to in or below `main()`) will only be run once when the app is compiled. In this case you'll end up with stale data pretty quickly.

Reply:
Got it. I'll fix.